src/components/CardGroup.tsx

```diff
@@ -38,10 +38,11 @@ interface Props {
   label: string;
   cards: Card[];
   onCardPressAction: (title: string) => void;
+  activeLabel?: string;
 }
 
-export const CardGroup = ({ label, cards, onCardPressAction }: Props) => {
-  const [choosenCard, setChoosenCard] = useState('');
+export const CardGroup = ({ label, cards, onCardPressAction, activeLabel }: Props) => {
```

Comment:
this component is too smart, and doesn't need to manage its own state, it can have props like this 
`{ label, cards, onCardPressAction, title }`
than you can delete complelty `const [choosenCard, setChoosenCard] = useState('');`
and pass title from redux like this:

<CardGroup
          onCardPressAction={title =>
            this.props.updateTransactionType(title === i18n.filterTransactions.received ? CONST.receive : CONST.send)
          }
          label={i18n.filterTransactions.transactionType}
          cards={[
            { title: i18n.filterTransactions.received, content: this.renderCardContent(i18n.filterTransactions.from) },
            { title: i18n.filterTransactions.sent, content: this.renderCardContent(i18n.filterTransactions.to) },
          ]}
          title={this.props.filters.transactionType}
        />

Reply:
But it won't be reusable anymore...

Reply:
I would say that it will be more reusable. The only difference with this solution is that it doesn't manage it's own state which can be problematic, the component remains dump and doesn't have knowledge about redux etc.
Image the situation that you have to make an additional button that also changes the filters.transactionType in redux, if the card component would be rendered already you would have to force the component CardGroup the remount (for example by setting key) to set the activeLabel again

Reply:
Oh, you are right. I misunderstood your point. Fixed.

Reply:
looks good:)